melodic/distribution.yaml

```
@@ -149,6 +149,35 @@ repositories:
       url: https://github.com/ubi-agni/agni_tf_tools.git
       version: master
     status: maintained
+  agxrobots_test:
+    doc:
+      type: git
+      url: https://github.com/885288421/agxrobots_test.git
+      version: 0.0.3
```

```suggestion
      version: master
```

This should be a branch name.